Gemfile

```diff
@@ -33,6 +33,8 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'dotenv-rails'
+  gem 'rest-client', '~> 2.1'
```

Comment:
Way did you add this gem on development and test env?
You will not use it on production env?

Reply:
The gem was moved to be used by all environments